source/optional_features/prefill_user_details/index.html.md.erb

```diff
@@ -41,12 +41,24 @@ All the parameters in `billing_address` are optional, and these parameters' valu
 
 If you include the `country` parameter, it must be an <a href="https://www.iso.org/obp/ui/#search/code/" target="_blank">ISO 3166-1 alpha-2 code</a>. If the parameter is invalid or missing, the __Country or territory__ field on the payment page will default to 'United Kingdom'.
 
-## Data collected by your GOV.UK Pay admin account
+## Data collected by GOV.UK Pay
 
-After your user completes their payment, your GOV.UK Pay account will collect your user's:
+If you prefill payment fields, the data collected by GOV.UK Pay depends on:
 
-- email address - even if you've disabled collecting your users' email addresses
+- your service settings in the [GOV.UK Pay admin tool](https://selfservice.payments.service.gov.uk/my-services)
+- where your user gets to on their payment journey
+
+By default, if your user completes their payment or your user completes the **Enter card details** page
+but does not complete their payment, GOV.UK Pay collects your user's:
```

Comment:
Can we remove a lot of this top section now as we're not explaining that it depends on where the user gets to in their payment journey anymore?